### DISCUSSION TITLE
Enable or disable endpoints on environments

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#123] [Ideas]
Hi!

I have a proposal about the endpoint section: currently, endpoints are applied to each environment. Is it planned to enable/disable endpoints based on a particular environment?

**Comment by davidebianchi** [comment]
Hi @giovf96! Yes, it is already planned as all the other sections in console design. The sections will be enabled if the related service are present in the project configuration to reduce unused part of Console which could bring confusion to our users.

**Comment by akelity** [comment]
This feature has received positive feedback and will be carefully evaluated for inclusion in the roadmap as soon as it becomes feasible.
